kalite/version.yaml

```diff
@@ -0,0 +1,397 @@
+
+"VERSION" :

```

Comment:
Hmm can you try removing the quotes in here and just have it as:

```yaml
0.13.0:
    - release_date:
    - git_commit: b9caa
```

If that works, I prefer that. It looks cleaner :) 

Also, I think you should remove the VERSION entry, since yaml doesn't do any substitution.